core/src/components/toast/toast.tsx

```diff
@@ -484,6 +493,16 @@ export class Toast implements ComponentInterface, OverlayInterface {
     return buttons;
   }
 
+  private getShape(theme: Theme): string | undefined {
+    const { shape } = this;
+
```

Comment:
Add a check here for if `theme !== 'ionic'` to return `undefined` as we don't support shapes in `ios` or `md` themes yet. Example: https://github.com/ionic-team/ionic-framework/blob/next/core/src/components/modal/modal.tsx#L904-L907

Reply:
I agree with Tanner. Make sure to add this check and also create the ticket.